chevah/compat/administration.py

```diff
@@ -41,19 +41,21 @@
 from chevah.compat.winerrors import ERROR_NONE_MAPPED
 
 
-def execute(
-    command, input_text=None, output=None, ignore_errors=True, verbose=False,
-        ):
+def execute(command, input_text=None, output=None, ignore_errors=True):
     """
     Execute a command having stdout redirected and using 'input_text' as
     input.
     """
+    verbose = False
```

Comment:
the verbose argument from execute was never used... this is more like an internal "debug" flag... which you enable it while investigating an issue